lib/content-services/src/lib/document-list/components/document-list.component.spec.ts

```diff
@@ -1567,6 +1584,56 @@ describe('DocumentList', () => {
         }), undefined);
     });
 
+    it('should display fileAutoDownload dialog if node size exceeds appConfig.preview-config.fileAutoDownloadSizeThresholdInMB', async () => {
+        appConfigService.config = {
+            ...appConfigService.config,
+            'preview-config': {
+                "enableFileAutoDownload": true,
+                "fileAutoDownloadSizeThresholdInMB": 10
+            }
+        }
+        documentList.navigationMode = DocumentListComponent.SINGLE_CLICK_NAVIGATION;
+        const node = { entry: {
+                ...mockNode1,
+                content: {
+                    ...mockNode1.content,
+                    sizeInBytes: 104857600
+                },
+            } };
+        documentList.onNodeClick(node);
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        expect(mockDialog.open).toHaveBeenCalledWith(FileAutoDownloadComponent, { disableClose: true });
+    });
+
+    it('should trigger file download when clicking on Download button on FileAutoDownload dialog', async () => {
+        mockDialog.open.and.returnValue({ afterClosed: () => of(FileAutoDownloadActionsEnum.DOWNLOAD) as any})
+        spyOn(nodeActionService, 'downloadNode');
+        appConfigService.config = {
+            ...appConfigService.config,
+            'preview-config': {
```

Comment:
move to document-list configuration

Reply:
Updated appConfig object name from "preview-config" to "viewer" as per [comment](https://github.com/Alfresco/alfresco-ng2-components/pull/8423/files#r1154214131)